eclipse-scout-core/src/table/TableAdapter.ts

```diff
@@ -651,6 +660,7 @@ export class TableAdapter extends ModelAdapter {
     }
 
     objects.replacePrototypeFunction(Table, '_createRow', TableAdapter._createRowRemote, true);
+    objects.replacePrototypeFunction(Table, 'replaceRows', TableAdapter.replaceRows, true);
```

Comment:
Why is this replaced for classic? This is only used by PageWithTable which does not exist in 
classic mode